packages/ai-bot/main.ts

```diff
@@ -142,26 +142,35 @@ function getResponse(history: DiscreteMatrixEvent[], aiBotUsername: string) {
   }
 }
 
+function maybeErrorMessage(error: any): string | undefined {
+  if (error instanceof OpenAIError) {
+    return `OpenAI error: ${error.name} - ${error.message}`;
+  }
+  if (typeof error === 'string') {
+    return `Unknown error: ${error}`;
+  }
+  return;
+}
+
 async function sendError(
   client: MatrixClient,
   room: Room,
   error: any,
   eventToUpdate: string | undefined,
 ) {
-  if (error instanceof OpenAIError) {
-    log.error(`OpenAI error: ${error.name} - ${error.message}`);
-  } else {
-    log.error(`Unknown error: ${error}`);
-  }
   try {
+    let errorMessage = maybeErrorMessage(error);
+    if (errorMessage) {
+      log.error(errorMessage);
+    }
     await sendMessage(
       client,
       room,
       'There was an error processing your request, please try again later',
       eventToUpdate,
       {
         isStreamingFinished: true,
-        errorMessage: error.message,
+        errorMessage,
```

Comment:
Downstream. Components rely on errorMessage to be available to display an message that has a red border. I wonder do we want to just default this to "Error"